### PR TITLE
Update Developer Guide for Remind command

### PR DESCRIPTION
**Pull request by edricyeo**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -276,12 +276,33 @@ The following activity diagram summarizes what happens when a user executes the
...
+
+Step 6: The `ReminderCommand` calls `Model#getFilteredPersonList` to get the filtered `List` of `Person` objects.
+
+Step 7: The `ReminderCommand` gets the `Person` to be edited from the `List`
+
+Step 8: The `ReminderCommand` creates a new edited `Person`, with the given `Reminder`
+
+Step 10: `ReminderCommand` then returns a new `CommandResult` with the result of the execution.
+
+![Sequence diagram for the Remind Command](images/ReminderSequenceDiagram.png)
```
Comment (tyw2811) [inline docs/DeveloperGuide.md:305]: I think you can add the limitation of PlantUML that causes the lifeline for ReminderCommandParser to continue even after the destroy marker (X).

**Review by tyw2811 — COMMENTED** [review]
Aside from the minor additions, LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (edricyeo) [reply docs/DeveloperGuide.md:305]: Thanks! Will update 🤩

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -276,12 +276,36 @@ The following activity diagram summarizes what happens when a user executes the
...
+
+Step 7: The `ReminderCommand` gets the `Person` to be edited from the `List`
+
+Step 8: The `ReminderCommand` creates a new edited `Person`, with the given `Reminder`
+
+Step 9: `ReminderCommand` then returns a new `CommandResult` with the result of the execution.
+
+![Sequence diagram for the Remind Command](images/ReminderSequenceDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```
[the remark this answers is not in the document]
Comment (edricyeo) [reply docs/DeveloperGuide.md]: Oops updated!

**Pull request merged**